apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/InvalidOrganizationExceptionTest.java

```diff
@@ -0,0 +1,20 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class InvalidOrganizationExceptionTest {
+
+  @Test
+  public void InvalidOrganizationException_ShouldReturnDetail_WhenDetailKeyIsPassed() {
+    // Set up the input detail
+    String detail = "Errordetail";
+
+    // Create an instance of InvalidOrganizationException
+    InvalidOrganizationException exception = new InvalidOrganizationException(detail);
+
+    // Verify the properties of the exception
+    assertThat(exception.getDetail()).isEqualTo(detail);
```

Comment:
Why test only for the details and not the status and type like BadRequestException?

Reply:
Because coverage had reached 100% lol

Reply:
Why do we need to test more properties in `BadRequestException` than in `InvalidOrganizationException`? Anyway, it's preferably to be consistent and comprehensive by testing thoroughly the output object (`exception`).

Reply:
I thought it'd be best not to perform unnecessary tests once coverage reached 100% so I removed the additional test to minimize the operations performed. I can add it in next week.